test/attributes/stun-address-attribute.js

```diff
@@ -10,29 +10,15 @@ test('encode', () => {
 
   const buf = attribute.toBuffer();
   const expectedBuffer = Buffer.from([
-    0 /* Reserved */,
-    0x1 /* Family */,
-    0xf8,
-    0x24 /* Port */,
-    0xc0,
-    0xa8,
-    1,
-    2 /* IP */,
+    0 /* Reserved */, 0x1 /* Family */, 0xf8, 0x24 /* Port */, 0xc0, 0xa8, 1, 2 /* IP */,
```

Comment:
This is a linter removing the line comments, making it difficult for a reader to scan.

Can you find ways to build up the array (perhaps in more steps, perf does not matter in this test, but legibility will matter) that the linter will accept, and that lets the code keep the by-line comments?